app/framework/store.js

```diff
@@ -170,10 +170,34 @@ function addLessonsToEvents(lessons) {
  * @param {string} language the active language
  */
 function filterEvents(events, language) {
-  // filter out events with wrong hostId
+  // filter out events with undesired parameters
+
+  // backwards compatibility fallback for single hostId filter 
   if (settings.hostIds instanceof Array) {
     events = events.filter(event => settings.hostIds.indexOf(event.HostId) !== -1);
   }
+  // or use initialListFilters array
+  else {
```

Comment:
```suggestion
  else if (settings.initialListFilters instanceof Object) {
```
@ursbraem Thanks. I have tested your feature and it works great! I think it's a good idea to not break old settings.

It is valid and to my knowledge common to set `settings.hostIds` to `null` so I'd change the `else` (see my suggestion) to avoid an Exception if `settings.initialListFilters` is not set.  

@schefbi [settings.js.example](https://github.com/erz-mba-fbi/kursausschreibung/blob/master/public/settings.js.example) will also get updated once the merge occurs.